packages/app/src/interfaces/bookmark-info.ts

```diff
@@ -29,4 +29,5 @@ export interface BookmarkFolderItems {
   name: string
   parent: string
   children: this[]
+  bookmarks: [BookmarkedPage]
```

Comment:
Could you match the notation and write it like this?
`bookmarkd: BookmarkedPage[]`